cost-analyzer/templates/cost-analyzer-checks-template.yaml

```diff
@@ -24,8 +24,17 @@ spec:
     metadata:
       labels:
         {{ include "cost-analyzer.commonLabels" . | nindent 8 }}
+        {{- if .Values.kubecostChecks.additionalLabels }}
+        {{ toYaml .Values.kubecostChecks.additionalLabels | nindent 8 }}
+        {{- end }}
     spec:
       template:
+        metadata:
+          labels:
+          {{ include "cost-analyzer.commonLabels" . | nindent 14 }}
+          {{- if .Values.kubecostChecks.additionalLabels }}
+          {{ toYaml .Values.kubecostChecks.additionalLabels | nindent 14 }}
+          {{- end }}
```

Comment:
Does the test include validation of this change?

Reply:
Yes-- see above. Do you mean the validation of the "job" template?

Reply:
Oh, yes, I believe that is what I meant. This section I highlighted applies to the pod, right? And that is covered by your test output. The other set of labels applies to the CronJob itself. The logic is the same, so it should be fine but I just didn't see it in the sample output.